spec/models/meeting_spec.rb

```diff
@@ -15,7 +15,7 @@ def make_groups_and_members!(groups, distributions)
 describe 'the cost of an individual meeting' do
   describe '#cost_from_shared_groups' do
     it 'calculates the right cost for a meeting of group members' do
-      groups = %w(      A      )
+      groups = %w( A      )
```

Comment:
Wasn't able to pass rubocop, it raised offense regarding trailing spaces after `(` 
Any idea?